content/applications/sales/crm/acquire_leads/lead_mining.rst

```diff
@@ -1,71 +1,75 @@
+For this feature to work, go to :menuselection:`CRM --> Configuration --> Settings` and activate
+the **Lead Mining** feature.
  
-.. image:: media/LM1.png
+.. image:: lead_mining/LM1.png
    :align: center
-
+   :alt: Activating Lead Mining
 
 Start generating leads
 ==========================
-You will now have a new button **Generate Leads** available in your pipeline.
-You are also able to create lead mining requests from the
-:menuselection:`Configuration --> Lead Mining Requests` and trough
+You will now have a new button **Generate Leads** available in your pipeline. Lead mining
+requests are also available from :menuselection:`Configuration --> Lead Mining Requests` or through
 :menuselection:`Leads --> Leads` where you have the **Generate Leads** button.
 
-.. image:: media/LM2.png
+.. image:: lead_mining/generate-leads-button.png
    :align: center
+   :alt: Generate Leads Buttons
 
+From there, click on the **Generate Leads** button, a window where you will be able to pick your
+criteria will pop up.
 
-From there, click on the **Generate Leads** button, a window where you will be able to pick your criteria will pop up.
-
-
-.. image:: media/LM5.png
+.. image:: lead_mining/generate-leads-popup.png
    :align: center
+   :alt: Generating Leads Pop-Up
 
+When choosing to target Companies and their contacts you can choose the contacts you are getting
+based on Role or Seniority. Make sure to be aware of the latest EU regulation when getting
+contact information. Get more information about General Data Protection Regulation on `Odoo GDPR
+<http://odoo.com/gdpr>`__,
 
-When choosing to target Companied and their contacts you can choose the contacts you are getting based on Role or Seniority. 
-When getting contact information make sure to be aware of the latest EU regulation, get more information about General Data Protection Regulation on
-`Odoo GDPR <http://odoo.com/gdpr>`__, 
-
-
-You can decide to filter the leads you are getting depending on the size (number of employees) of the companies.
-You can pick the countries your leads are coming from. It is possible to pick multiple countries.
-You can pick the industries your leads are coming from. It is possible to pick multiple industries.
-
-You can choose the salesperson and the Salesteam the lead will be assigned to. 
-You can also add tags to track your generated leads.
+Additional options you can choose from are:
+* **Size**: filter leads depending on the number of employees in the companies.
+* **Country**: pick the countries leads are coming from. Pick multiple countries, if applicable.
+* **Industry**: pick the industries leads are coming from. Pick multiple industries, if applicable.
+* **Salesperson**: choose who on the Sales Team will be assigned these leads.
+* **Sales Team**: choose which Sales Team will be assigned to these leads
+* **Tags**: choose what tags are added directly to the leads once found.
```

Comment:
```suggestion
Additional options you can choose from are:

- **Size**: filter leads depending on the number of employees in the companies.
- **Country**: pick the countries leads are coming from. Pick multiple countries, if applicable.
- **Industry**: pick the industries leads are coming from. Pick multiple industries, if applicable.
- **Salesperson**: choose who on the Sales Team will be assigned these leads.
- **Sales Team**: choose which Sales Team will be assigned to these leads
- **Tags**: choose what tags are added directly to the leads once found.
```
I assume you wanted to have a list here? I think you need to have an empty line before starting a list. I never used "*", that's why I replaced them with "-", but "*" should work as well.